### ISSUE TITLE
Organization of Docs in Cards on 'Microsoft 365 Community Content'

### ISSUE THREAD

**Issue by sympmarc** [MicrosoftDocs/microsoft-365-community#157]
From @veronicageek:

Looking at the M365 Community Docs, I was wondering if maybe, we could put the articles in alphabetical order or something inside of their group (Basics, IA, etc…)? 
I think it would be easier… Knowing that when you’re on that page and you search, it’s actually not scoped to the Community Docs 😕

**Comment by sympmarc** [comment]
Yes, it is going to get confusing over time. Talking to the MS folks, the goal of the “cards” is to highlight up to 10 articles per category. There will be more articles available on the left side after a while.
 
To date, I’ve been adding articles to the bottom of the cards as they come in. That makes little sense, of course. Alphabetical probably doesn’t make a lot of sense, either, since the title can be worded in many ways. I’m thinking in descending order of “popularity”, but @JoanneHendrickson  can’t pull those stats just yet.

**Comment by veronicageek** [comment]
Thanks @sympmarc - It's just to give a little bit of organisation or make it easier for users. I think it could bring ideas and requests from users for more topics 🙂

**Comment by sympmarc** [comment]
In my mind, showing the "Top 10" in each card makes sense, with everything else still available on the left. Over time, I'm sure our thinking will evolve based on the activity we see, plus the way the corpus of content expands.

**Comment by JoanneHendrickson** [comment]
I agree Veronica, we need to have easy to navigate experience especially if the content set expands into all aspects of M365. I am gathering what data there is at this stage, and soon we will have reports generated for this repo.
